support-models/src/test/scala/com/gu/support/encoding/PaymentMethodEncoderSpec.scala

```diff
@@ -26,7 +26,7 @@ class PaymentMethodEncoderSpec extends AsyncFlatSpec with Matchers with LazyLogg
       "StreetNumber" : null,
       "BankTransferType" : "DirectDebitUK",
       "Type" : "BankTransfer",
-      "paymentGateway": "GoCardless"
+      "PaymentGateway": "GoCardless"
```

Comment:
was this wrong before?

Reply:
Yes! The tests were passing anyway because the `paymentGateway` field in the case classes had default values, so decoding still succeeded!

Reply:
oh dear! good spot.